### ISSUE TITLE
Brand new setup...

### ISSUE THREAD

**Issue by tivomaniac** [hoobs-org/HOOBS#749]
Connecting via WiFi. I got logged into the HOOBS wifi and entered my network info and password but when I try to connect to http://hoobs.local I get hoobs.local's server IP address could not be found. I logged into my router and HOOBS is connected so I tried to connect via the IP address to see if that would work but that doesn't work either.  What's next?

**Comment by askovi** [comment]
If you're using a mesh network, you will have to connect the HOOBS Box into the main node for it to be accessible across the network. 

If you're using Windows, you must have the Apple Bonjour Service installed (restart your computer after installing it): https://support.apple.com/kb/DL999?locale=en_US

HOOBS is also available at the following addresses:

http://hoobs (no domain)

http://hoobs.lan (some routers)

http://192.168.x.x (replace with local ip address given to the HOOBS Box by your router)

**Comment by tivomaniac** [comment]
No mesh network here.

Installed the Apple Bonjour Service and rebooted. I've tried:

http://hoobs.local
http://hoobs
http://hoobs.lan

For all 3 of those I get hoobs’s server IP address could not be found.

http://192.168.50.36 (which is the IP address it is on) for that Chrome shows connecting for a bit and then times out to "took too long to respond".  Before I loaded Bonjour it showed connection refused when I tried the IP address.

I can try connecting directly to the router if it's not too late for that already.
